packages/data-context/src/data/ProjectLifecycleManager.ts

```diff
@@ -252,13 +260,13 @@ export class ProjectLifecycleManager {
             finalConfig.port = 4455
           }
 
-          span?.end()
-
           if (!devServerOptions?.port) {
             throw getError('CONFIG_FILE_DEV_SERVER_INVALID_RETURN', devServerOptions)
           }
 
           finalConfig.baseUrl = `http://localhost:${devServerOptions?.port}`
+
+          span?.end()
```

Comment:
if line 264 gets hit, this span doesn't end - what effect does an orphan span have on the telemetry for `dataContext:ct:startDevServer` ?

Reply:
the spans look super messy. Ill fix this. Thank you for catching this!

Reply:
addressed in e4eaaf8986